### ISSUE TITLE
how to rename a tab ?

### ISSUE THREAD

**Issue by BarryArch** [atinylittleshell/TerminalOne#41]
It seems that tabs can not be renamed, it just named by numbers

**Comment by atinylittleshell** [comment]
yeah currently tabs can't be named. Is this an important feature to you? Would you like for them to automatically named by some rules or is it more useful to you to support manual rename?

**Comment by jjaimealeman** [comment]
This is one of the reasons I refuse to use Alacritty and why I chose Kitty - **tabs**.

Added benefit is being able to rename tabs and move tab order.

One tab may have Ranger, another neovim in one project, another tab for another neovim project, another running htop ... 

So I too would love to have tab-naming ability :D
